index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <title>PortfolioPage</title>
+  </head>
+  <body>
+    <header>
+      <div class="logo-placeholder"><a href="#">Rich</a></div>
```

Comment:
Place your logo inside the `nav` tag as this is semantically correct
- Refer to this article for more [information](https://code.tutsplus.com/tutorials/30-css-best-practices-for-beginners--net-6741)